README.md

```diff
@@ -107,67 +163,104 @@ that runs at a much lower price than normal instances. However, Compute
 Engine might terminate (preempt) these instances if it requires access 
 to those resources for other tasks; default: `false`
 
-### `public_key_path`
+### `service_account_name`
 
-Path to the public half of the ssh key that will be deployed to
-`~username/.ssh/authorized_keys`; see also [`username`](#username) below.
+The name of the service account to use when enabling account scopes. This
+is usually an email-formatted service account name created via the "Permissions"
+tab of the GCP console.
 
-### `region`
+This is ignored unless you specify any `service_account_scopes`.
+
+Default: "default"
+
+### `service_account_scopes`
+
+An array of scopes to add to the instance, used to grant additional permissions
+within GCP.
 
-Region in which to launch instances.  `region` is defined as the part
-prior to the second hyphen in an availability zone's name; e.g. in
-`us-central1-b`, the region is `us-central1`.  Specifying region but
-not `zone_name` allows kitchen-gce to avoid launching instances into a
-zone that is down for maintenance.  If `any` is specified, kitchen-gce
-will select a zone from all regions.  Default: `us-central1` (lowest
-cost region); valid values: `any`, `asia-east1`, `europe-west1`,
-`us-central1`
+The scopes can either be a full URL (i.e. `https://www.googleapis.com/auth/devstorage.read_write`)
+or the short-name (i.e. `devstorage.read_write`), but aliases (i.e. `storage-rw`) are not permitted.
```

Comment:
Feature request heard loud and clear. I will update this branch with that before we decide to merge it, just for you. :)